### PR TITLE
[Fix] User dropdown tabbing

### LINKED ISSUE #1011: [Fix] Username dropdown tabbing
**Describe the bug**
- Need to instantly tab down to dropdown elements.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to the search page.
2. Start tabbing to test.

**Expected behaviour**
- Instantly tab down to elements once landed on the dropdown.

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by SukhsimranpreetSekhon**
#### Changes introduced

- Fixed tabbing of elements in username dropdown on all pages.

#### Related issue(s)

closes #1011 

#### Checklist
If the database has been modified:
- [ ] Update database diagram 
- [ ] Create new migration 

If API endpoints has been modified:
- [ ] Update backend documentation 
- [ ] Create/Update validators for the API endpoints 
- [ ] The API endpoints are properly secured with keycloak 

If the UI has been modified:
- [x] It is translated in both language (with no hard coded text) 
- [ ] Has been tested on IE
- [x] The modifications are tab friendly
- [x] It is accessible

### CODE REVIEW

**Comment by SukhsimranpreetSekhon** [comment]
Not fully working yet. Some reason still has to tab over the change language button and then press tab once more to get onto the dropdown. Figuring out why.

**Comment by BenJeau** [comment]
image

Just tried this on your branch, and it goes directly to the menu (but there still that tab twice bug, but I think its fine)

Peek 2020-10-27 13-21

**Comment by SukhsimranpreetSekhon** [comment]
@BenJeau Yeah, it is perfect. Just going to delete the SCSS file changes I added by accident.

**Review by BenJeau — APPROVED** [review]

**Comment by SukhsimranpreetSekhon** [comment]
> @BenJeau Yeah, it is perfect. Just going to delete the SCSS file changes I added by accident.

Nevermind, it is fixed now.

File: services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx [hunk 1]
```diff
@@ -47,7 +47,11 @@ const TopNavView = ({ isAdmin, loading, displaySearch, displayLogo, intl }) => {
 
   // menu options for profile dropdown
   const menu = (isDropdown, optionalStartMenuItems) => (
-    <Menu className={isDropdown ? "dropDownMenu" : "hamburgerMenu"}>
+    <Menu
+      tabIndex={-1}
+      tabindex="-1"
```
Comment (BenJeau) [inline services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx:55]: I would maybe put a small comment here explaining why this is done, and you'll need to fix the linting errors (https://cicd.ised-isde.canada.ca/blue/organizations/jenkins/DSD%2FI-Talent%2FI-Talent/detail/user-dropdown-accessibility/9/pipeline), for this one here, you'll just need to disable eslint for these two lines for the `react/jsx-no-duplicate-props` rule. Maybe something like `/* eslint-disable react/jsx-no-duplicate-props */` then this afterwards `/* eslint-enable react/jsx-no-duplicate-props */`

Comment (SukhsimranpreetSekhon) [reply services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx:55]: Tab index at -1 is used for off-screen content that appears on a specific event. The element is not reachable via sequential keyboard navigation, however, we focused the dropdown with Javascript instead so it works for tabbing and normal clicking.

**Review by dynamic11 — APPROVED** [review]

**Comment by dynamic11** [comment]
there is still a lint error. if you attach shell to frontend and run "yarn lint" you can see the errors locally
image

**Comment by SukhsimranpreetSekhon** [comment]
> there is still a lint error. if you attach shell to frontend and run "yarn lint" you can see the errors locally
> image

Fixed.

**Comment by BenJeau** [comment]
@SukhsimranpreetSekhon feel free to merge it

**Pull request merged**